src/CribblyBackend/Controllers/TeamController.cs

```diff
@@ -59,5 +59,25 @@ public async Task<IActionResult> Create([FromBody] Team team)
                 return StatusCode(500, $"Uh oh, bad time: {e.Message}");
             }
         }
+        /// <summary>
+        /// GetAll returns all teams that are in the active tournament. 
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("GetAll")]
```

Comment:
:book: It's more typical in a REST API that sending a `GET` request to an endpoint without an ID specified returns all of that entity; for example, it would be typical that sending a `GET` to `/api/teams` would implicitly return all of the teams, whereas sending something a `GET` to `/api/teams/123` would get the team with ID 123.

Reply:
I was wondering what to call this. Fixed